app/views/admin/ambassadors/show.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1= "Ambassador: #{@ambassador.username}"
+
+- if @ambassador.avatar
+  - unless @ambassador.avatar.url(:medium) == 'https://files.bikeindex.org/blank.png'
+    = image_tag @ambassador.avatar.url(:medium), class: 'users-show-avatar'
+
+
+%table.table.table-striped.table-hover.table-bordered.table-sm.without-exterior-border
+  %tbody
+    - @ambassador.attributes.slice(*@user_attributes).each_pair do |key, value|
+      %tr
+        %td
+          %strong= key
+        %td= value
+
+%h2 Ambassador Tasks
+
+%p
+  %strong= "#{@ambassador.progress_count} complete"
+
+%table.table.table-striped.table-hover.table-bordered.table-sm.without-exterior-border.ambassador-tasks-table
+  %thead.small-header
+    %tr
+      %th Task
+      %th Status
+  %tbody
+    - @ambassador.ambassador_task_assignments.each do |task|
+      %tr
+        %td= task.description_html.html_safe
+        %td= task.status
```

Comment:
Rather than status here, it would be nice to get a completed at date.

There is a handy way of displaying this:

```haml
- if task.completed_at
  %span.convertTime
    = l task.completed_at, format: :convert_time
```